releasenotes/notes/fix-logs-unschedule-multi-source-config-765449fb8a43fee6.yaml

```diff
@@ -0,0 +1,6 @@
+---
+fixes:
+  - |
+    This fixes a regression introduced in `7.36.0` where some logs sources would not be unschedule if they had been
+    created from multi-source config. This would lead to duplicate log entries being created as there would
+    be more than one tailer tailing the targeted source.
```

Comment:
```suggestion
    This fixes a regression introduced in ``7.36.0`` where some logs sources attached to a container/pod would not be unscheduled on
    container/pod stop if multiple logs configs were attached to the container/pod.
    This could lead to duplicate log entries being created on container/pod restart as there would
    be more than one tailer tailing the targeted source.
```